server/lib/chat/chat-service.ts

```diff
@@ -217,27 +206,40 @@ export default class ChatService {
         "Can't moderate yourself",
       )
     }
+
+    const { channelPermissions: moderatorChannelPermissions } = moderatorUserChannel
+    const { channelPermissions: targetChannelPermissions } = targetUserChannel
+
+    // TODO(2Pac): Really need tests for these.
+
+    if (targetChannelPermissions.owner && !moderatorPermissions?.moderateChatChannels) {
+      throw new ChatServiceError(
+        ChatServiceErrorCode.ModeratorAccess,
+        'Only global moderators can moderate channel owners',
```

Comment:
I don't think `global moderator` is a good term to standardize on, it seems to imply something in a geographic sense. `Server moderator` or `Service moderator` or `ShieldBattery staff` would be better imo